Jenkinsfile

```diff
@@ -31,6 +31,8 @@ pipeline {
                   /usr/share/maven/bin/mvn \
                       clean verify -B -V -X -e \
                       -Dmaven.test.failure.ignore=true \
+                      -P baseline-compare-and-replace \
+                      -P api-baseline-check \
```

Comment:
I think something similar should be added to https://github.com/eclipse-cdt/cdt-lsp/blob/b12dbaf55758c4e190ce9833131ea47a9e6bb94b/.github/workflows/tycho-build.yaml#L30

For CDT it is done [here](https://github.com/eclipse-cdt/cdt/blob/75521e0e9fb73975a24bbe98581dc6a8f56dee05/releng/scripts/check_bundle_versions.sh#L33-L34) which is called by the workflow [here](https://github.com/eclipse-cdt/cdt/blob/2c3aaf89a1996957c2574ca9f9f2f1076310352e/.github/workflows/code-cleanliness.yml#L33-L36) It is more complicated in CDT side because it is so many more bundles that we use a wrapper script to handle it and run it in parallel with the test jobs. That extra complication isn't needed in cdt-lsp, so just adding the new profiles to the tycho-build.yaml seems sufficient to me